mantidimaging/gui/test/gui_system_base.py

```diff
@@ -48,6 +48,10 @@ def tearDown(self) -> None:
             leak_tracker.pretty_print(debug_init=False, debug_owners=False, trace_depth=5)
             leak_tracker.clear()
 
+        for widget in self.app.topLevelWidgets():
```

Comment:
Yes, that's correct. This would definitely not be the desired long term solution - more just a suggestion as a temporary measure while the problem is being looked into if we know that the test itself should pass otherwise. As such, very happy to leave in it's current state also 🙂